recipes/lammps/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "lammps" %}
+{% set version = "2018.03.16" %}
+{% set sha256 = "03f91d69f472c37a4008611dee13fad91b6bedc66dbf0393416cdacc751b57ff" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: stable_16Mar2018.tar.gz
+  url: https://github.com/lammps/lammps/archive/stable_16Mar2018.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - toolchain
+    - setuptools
+    - gcc 
```

Comment:
can you switch this to the new style compilers

```
build:
  - {{ compiler('c') }}
  - {{ compiler('cxx') }}
  - {{ compiler('fortran') }}

```
you can remove gcc and toolchain

Reply:
@mariusvniekerk  I put them in as otherwise I get the following warnings:

```
WARNING (lammps,bin/lmp_serial): /lib64/libdl.so.2 not found in sysroot, is this binary repackaging? .. do you need to use install_name_tool/patchelf?
   INFO (lammps,bin/lmp_serial): Needed DSO lib/libpython2.7.so.1.0 found in conda-forge::python-2.7.14-5
WARNING (lammps,bin/lmp_serial): /lib64/libc.so.6 not found in sysroot, is this binary repackaging? .. do you need to use install_name_tool/patchelf?
WARNING (lammps,bin/lmp_serial): Needed DSO lib/libgfortran.so.3 found in ['libgcc', 'gcc']
WARNING (lammps,bin/lmp_serial): .. but ['libgcc', 'gcc'] not in reqs/run, i.e. it is overlinked (likely) or a missing dependency (less likely)
WARNING (lammps,bin/lmp_serial): Needed DSO lib/libgcc_s.so.1 found in ['libgcc-ng', 'gcc']
WARNING (lammps,bin/lmp_serial): .. but ['libgcc-ng', 'gcc'] not in reqs/run, i.e. it is overlinked (likely) or a missing dependency (less likely)
WARNING (lammps,bin/lmp_serial): Needed DSO lib/libstdc++.so.6 found in ['gcc', 'libstdcxx-ng']
WARNING (lammps,bin/lmp_serial): .. but ['gcc', 'libstdcxx-ng'] not in reqs/run, i.e. it is overlinked (likely) or a missing dependency (less likely)
WARNING (lammps,bin/lmp_serial): Needed DSO lib/libz.so.1 found in ['zlib']
WARNING (lammps,bin/lmp_serial): .. but ['zlib'] not in reqs/run, i.e. it is overlinked (likely) or a missing dependency (less likely)
WARNING (lammps,bin/lmp_serial): /lib64/libpthread.so.0 not found in sysroot, is this binary repackaging? .. do you need to use install_name_tool/patchelf?
WARNING (lammps,bin/lmp_serial): /lib64/libutil.so.1 not found in sysroot, is this binary repackaging? .. do you need to use install_name_tool/patchelf?
WARNING (lammps,bin/lmp_serial): /lib64/libm.so.6 not found in sysroot, is this binary repackaging? .. do you need to use install_name_tool/patchelf?
WARNING (lammps,lib/python2.7/site-packages/liblammps.so): /lib64/libdl.so.2 not found in sysroot, is this binary repackaging? .. do you need to use install_name_tool/patchelf?
WARNING (lammps,lib/python2.7/site-packages/liblammps.so): /lib64/libc.so.6 not found in sysroot, is this binary repackaging? .. do you need to use install_name_tool/patchelf?
WARNING (lammps,lib/python2.7/site-packages/liblammps.so): Needed DSO lib/libstdc++.so.6 found in ['gcc', 'libstdcxx-ng']
WARNING (lammps,lib/python2.7/site-packages/liblammps.so): .. but ['gcc', 'libstdcxx-ng'] not in reqs/run, i.e. it is overlinked (likely) or a missing dependency (less likely)
WARNING (lammps,lib/python2.7/site-packages/liblammps.so): Needed DSO lib/libgcc_s.so.1 found in ['libgcc-ng', 'gcc']
WARNING (lammps,lib/python2.7/site-packages/liblammps.so): .. but ['libgcc-ng', 'gcc'] not in reqs/run, i.e. it is overlinked (likely) or a missing dependency (less likely)
WARNING (lammps,lib/python2.7/site-packages/liblammps.so): Needed DSO lib/libz.so.1 found in ['zlib']
WARNING (lammps,lib/python2.7/site-packages/liblammps.so): .. but ['zlib'] not in reqs/run, i.e. it is overlinked (likely) or a missing dependency (less likely)
   INFO (lammps,lib/python2.7/site-packages/liblammps.so): Needed DSO lib/libpython2.7.so.1.0 found in conda-forge::python-2.7.14-5
WARNING (lammps,lib/python2.7/site-packages/liblammps.so): Needed DSO lib/libgfortran.so.3 found in ['libgcc', 'gcc']
WARNING (lammps,lib/python2.7/site-packages/liblammps.so): .. but ['libgcc', 'gcc'] not in reqs/run, i.e. it is overlinked (likely) or a missing dependency (less likely)
WARNING (lammps,lib/python2.7/site-packages/liblammps.so): /lib64/libpthread.so.0 not found in sysroot, is this binary repackaging? .. do you need to use install_name_tool/patchelf?
WARNING (lammps,lib/python2.7/site-packages/liblammps.so): /lib64/libutil.so.1 not found in sysroot, is this binary repackaging? .. do you need to use install_name_tool/patchelf?
WARNING (lammps,lib/python2.7/site-packages/liblammps.so): /lib64/libm.so.6 not found in sysroot, is this binary repackaging? .. do you need to use install_name_tool/patchelf?
```

Reply:
What version of conda-build are you using?

Reply:
@conda-forge/core This seemed to work with toolchain, but not with the new compiler syntax, thoughts?